### PR TITLE
Initial CHANGELOG.md update for 21.08

### PR DESCRIPTION
**Pull request by sameerz**
Signed-off-by: Sameer Raheja <sraheja@nvidia.com>

### CODE REVIEW

**Comment by sameerz** [comment]
build

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,263 @@
...
+### Performance
+|||
+|:---|:---|
+|[#866](https://github.com/NVIDIA/spark-rapids/issues/866)|[FEA] combine window operations into single call|
+|[#2800](https://github.com/NVIDIA/spark-rapids/issues/2800)|[FEA] Support ORC small files coalescing reading|
+|[#737](https://github.com/NVIDIA/spark-rapids/issues/737)|[FEA] handle peer timeouts in shuffle|
+|[#1590](https://github.com/NVIDIA/spark-rapids/issues/1590)|Rapids Shuffle - UcpListener|
+|[#2275](https://github.com/NVIDIA/spark-rapids/issues/2275)|[FEA] UCP error callback deal with cleanup|
+|[#2799](https://github.com/NVIDIA/spark-rapids/issues/2799)|[FEA] Support ORC multi-file cloud reading|
+|[#2334](https://github.com/NVIDIA/spark-rapids/issues/2334)|[FEA] Record GPU time and Fetch time separately, instead of recording Total Time|
```
Comment (jlowe) [inline CHANGELOG.md:34]: This does not impact performance, this is a change in how metrics are reported.

Comment (sameerz) [reply CHANGELOG.md]: Moved 2334 from the Performance section to Feature

**Review by jlowe — APPROVED** [review]

**Comment by sameerz** [comment]
build

**Pull request merged**